man/systemd.network.xml

```diff
@@ -796,6 +796,48 @@
       </variablelist>
   </refsect1>
 
+    <refsect1>
+    <title>[RoutingPolicyRule] Section Options</title>
+
+      <para>An <literal>[RoutingPolicyRule]</literal> section accepts the
+      following keys. Specify several <literal>[RoutingPolicyRule]</literal>
+      sections to configure several rules.</para>
+
+      <variablelist class='network-directives'>
+        <varlistentry>
+          <term><varname>TypeOfService=</varname></term>
+          <listitem>
+            <para>Specifies the type of service to match.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>From=</varname></term>
+          <listitem>
+            <para>Specifies the source prefix to match. Possibly followed by a slash and the prefix length.</para>
```

Comment:
this should say "address prefix" instead of just "prefix", otherwise it's not clear at all what this is about